### DISCUSSION TITLE
What is the meaning of transparency in the background settings?

### DISCUSSION THREAD

**Discussion by saschamarco** [freeplane/freeplane#2056] [General]
Hello, i am new to freeplane and still exploring all the options.

The background settings allow to adjust the transparency, but i do not understand what is supposed to become transparent in relation to what.
Is the mindmap background supposed to show other programs / windows running "behind" freeplane?
Is it what the changelog of 1.12.2 refers to as "Support translucent backgrounds for mind maps"?

I played around with this setting and it makes my background more or less greyish, but i am using linux where maybe this feature does not work. I currently use the desktop-compositor to make freeplane slightly transparent with a video of a calming scenery playing in the background (omg that's kinda awkward, lol), but of course that also makes the nodes, text, UI etc. just as transparent, limiting visibility.

Thank you and have a nice day!

**Comment by euu2021** [comment]
This is the discussion that explains transparent background: https://github.com/freeplane/freeplane/issues/1077
